src/orders/Dockerfile

```diff
@@ -63,7 +61,8 @@ EXPOSE 80
 WORKDIR /app
 ENV APPLICATION_NAME=${APPLICATION_NAME}
 ENV VERSION=${VERSION}
-ENV TNS_ADMIN=/usr/lib/oracle/${clientVersion}/client64/lib/network/admin/
+# Edit the helm chart to map
```

Comment:
"Edit the helm chart to map" ?

Reply:
This is just a comment to users who want to dig a bit deeper to map the wallets themselves etc. Is the comment that this is too vague ?(I agree its valgue) . Shall I make it a bit more explicit and spell it out ?

Reply:
Remove comment and put a note somewhere. It's not common for users to mess around wallets. I cannot even see a use case.

Reply:
removed.